src/content/developers/docs/scaling/layer-2-rollups/index.md

```diff
@@ -87,7 +87,7 @@ Multiple implementations of Optimistic rollups exist that you can integrate into
 
 ### Zero-knowledge rollups {#zk-rollups}
 
-**Zero-knowledge rollups (ZK-rollups)** bundle (or "roll-up") hundreds of transfers off-chain and generate a cryptographic proof, known as a SNARK (succinct non-interactive argument of knowledge). This is known as a validity proof and is posted on layer 1.
+**Zero-knowledge rollups (ZK-rollups)** bundle (or "roll-up") hundreds of transfers off-chain and generate a cryptographic proof. These proofs can come in a form of a SNARK (Succinct Non-interactive ARgument of Knowledge) or a STARK (Scalable Transparent ARgument of Knowledge). This is known as a validity proof and is posted on layer 1.
```

Comment:
- I don't think we want to be so blunt about where the acronyms come from. It seems messy and doesn't add much in terms of understanding.
- Small grammar fix

```suggestion
**Zero-knowledge rollups (ZK-rollups)** bundle (or "roll-up") hundreds of transfers off-chain and generate a cryptographic proof. These proofs can come in the form of SNARKs (succinct non-interactive argument of knowledge) or STARKs (scalable transparent argument of knowledge). SNARKs and STARKs are known as validity proofs and get posted to layer 1.
```